docs: add show user functions and system.user_functions (#447)
datafuselabs/databend-docs: docs/sql-reference/10-sql-commands/50-administration-cmds/show-functions.md

---
title: SHOW FUNCTIONS
---

import FunctionDescription from '@site/src/components/FunctionDescription';

<FunctionDescription description="Introduced or updated: v1.2.315"/>

> Shows the list of supported builtin scalar/aggregate functions currently.

Reviewer: ```suggestion
Lists the currently supported built-in scalar and aggregate functions.
```
